app/models/application_record.rb

```diff
@@ -1,3 +1,42 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def set_bad_method
```

Comment:
Bliss.AI: report

| Debt | Issue |
|---:|---|
| 8.0 | Perceived complexity for set_bad_method is too high. [19/7] |
| 7.5 | Cyclomatic complexity for set_bad_method is too high. [17/6] |
| 2.0 | Method has too many lines. [30/10] |
| 0.5 | Assignment Branch Condition size for set_bad_method is too high. [19.6/15] |